### PR TITLE
Punktuation von Optionen entfernt

### PR DESCRIPTION
**Pull request by Zack-83**

### CODE REVIEW

**Review by thrichter-FD — CHANGES_REQUESTED** [review]
Small detail that can be easily changed: typo in line 732 - "following" rather than "followig". Else fine, while I didn't check all languages.

File: rdmorganiser/options/rdmo.xml [hunk 1]
```diff
@@ -729,11 +729,11 @@
 		<dc:comment/>
 		<optionset dc:uri="https://rdmorganiser.github.io/terms/options/dataset_license_options"/>
 		<order>0</order>
-		<text lang="en">The dataset will be published under the followig license:</text>
-		<text lang="de">Der Datensatz wird unter folgender Lizenz veröffentlicht:</text>
-		<text lang="es">El conjunto de datos se publicará bajo la siguiente licencia:</text>
-		<text lang="fr">Le jeu de données sera publié sous la licence suivante&#8239;:</text>
-		<text lang="it">La raccolta dati verrà pubblicata con la seguente licenza:</text>
+		<text lang="en">The dataset will be published under the followig license</text>
```
Comment (Zack-83) [inline rdmorganiser/options/rdmo.xml:732]:
```suggestion
		<text lang="en">The dataset will be published under the following license</text>
```

**Comment by Zack-83** [comment]
> Small detail that can be easily changed: typo in line 732 - "following" rather than "followig". Else fine, while I didn't check all languages.

Thanks, Done https://github.com/rdmorganiser/rdmo-catalog/pull/183/commits/4034e71734b4f8f453826bc366209474ff2bf0dc

**Review by thrichter-FD — APPROVED** [review]
Now everything seems fine

**Pull request merged**